packages/ui-components/src/data-display/Card/Card.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react';
+import { Image } from '../Image'
+import { Button } from '../../input-controls/Button'
+
+type Props = {
+  title?: string;
+  body?: string;
```

Comment:
i'm not sure what this component will be used for, but think about making these (at least body) any, or jsx element
so we can provide custom body, not only string

Reply:
Nice suggestion :smile: